SimpleTotp/Base32Convert.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+
+namespace SimpleTotp
+{
+    /// <summary>
+    /// Converts to/from Base32
+    /// </summary>
+    /// <remarks>
+    /// <see href="https://en.wikipedia.org/wiki/Base32"/>
+    /// </remarks>
+    public static class Base32Convert
+    {
+        /// <summary>
+        /// Length of the block in the Base32 conversion algorithm
+        /// </summary>
+        private const int BlockLength = 5;
+
+        /// <summary>
+        /// Number of bits in byte (duh!) - used for readability only
+        /// </summary>
+        private const int BitsInByte = 8;
+
+        /// <summary>
+        /// Base32 padding character
+        /// </summary>
+        private const char PaddingChar = '=';
```

Comment:
Why this character?

Reply:
http://www.herongyang.com/Encoding/Base32-Encoding-Algorithm.html